database/pgtsv_to_json

```diff
@@ -135,14 +139,25 @@ def main():
   # See: http://grokbase.com/t/python/python-ideas/131b0eaykx/csv-dialect-enhancement
   # See: http://stackoverflow.com/questions/11379300/python-csv-reader-behavior-with-none-and-empty-string
   # See: https://github.com/JoshClose/CsvHelper/issues/252
-  reader = unicode_csv_reader(sys.stdin, delimiter='\t', quotechar=None, quoting=csv.QUOTE_NONE)
+  reader = unicode_csv_reader(in_file, delimiter='\t', quotechar=None, quoting=csv.QUOTE_NONE)
   for line in reader:
     obj = {}
     for (name, convert), field in zip(names_converters, line):
-        obj[name] = None if field == "\\N" \
+        obj[name] = None if field in NULL_STRINGS \
```

Comment:
I think only `\N` should be treated as None at the top level?  `NULL`s are None only within arrays.